test/e2e/README.md

```diff
@@ -192,6 +192,41 @@ You can load the new image into the kind cluster using the command below:
 kind load docker-image projects.registry.vmware.com/antrea/antrea-ubuntu:latest --name <kind_cluster_name>
 ```
 
+## Running the e2e tests on an existing K8s cluster
+
+If you already have a K8s cluster on which Antrea has been deployed, the following steps should be done for running the e2e tests.
+
+First, you should config the ssh information for each Node in the cluster. Here is an example:
+
+```bash
+Host <Control-Plane-Node>
+    HostName <Control-Plane-IP>
+    Port 22
+    user ubuntu
+    IdentityFile /home/ubuntu/.ssh/id_rsa
+Host <Worker-Node>
+    HostName <Worker-Node-IP>
+    Port 22
+    user ubuntu
+    IdentityFile /home/ubuntu/.ssh/id_rsa
+```
+
+Make sure `Host` of each Node must be the same as the corresponding Node name. The `Port` is the default port of ssh service. As for the `IdentityFile`, add the location of the private key (`id_rsa` in example) to it.
+
+Besides the configuration of the ssh connection, make sure you have added the public key to `authorized_keys` of each Node and set `PubkeyAuthentication` and `RSAAuthentication` of ssh service to `yes`.
+
+Second, when running the e2e tests, kubeconfig of the cluster will be used, so make sure you have copied it to the right location, e.g. `$HOME/.kube/config` or the path specified by `-remote.kubeconfig`.
+
+Third, considering the e2e test will check whether Antrea has been deployed to the cluster, the `antrea.yml` should be put under the `/home/<user>` directory of the control-plane Node.
```

Comment:
> considering the e2e test will check whether Antrea has been deployed to the cluster

this is incorrect, rather: "the e2e test framework will ensure that Antrea has been deployed to the cluster by applying the deployment manifest"